motivaa-backend/motivaa-app/src/main/java/com/motivaa/control/repository/Elasticsearch.java

```diff
@@ -21,20 +20,20 @@
 
 @Service
 @Log4j2
-public class ESClient {
+public class Elasticsearch {
         @Value("${elasticsearch.host}")
         String esHost;
         RestClient restClient;
         ElasticsearchTransport transport;
-        ElasticsearchClient esClient;
+        co.elastic.clients.elasticsearch.ElasticsearchClient esClient;
```

Comment:
Solar complained that the variable for esClient is matching with the name of the class and it is a bad practice

Reply:
I understand why you changed your class name  EsClient to Elasticsearch. What I don't understand is why you use co.elastic.clients.elasticsearch.ElasticsearchClient instead of ElasticsearchClient.